### PR TITLE
upstream `tgw`

### PR DESCRIPTION
**Pull request by Benbentwo**
## what
* Upstream TGW new submodules

## why
* Cross region support

### CODE REVIEW

File: modules/tgw/cross-region-hub-connector/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| <a name="output_aws_ec2_transit_gateway_peering_attachment_id"></a> [aws\_ec2\_transit\_gateway\_peering\_attachment\_id](#output\_aws\_ec2\_transit\_gateway\_peering\_attachment\_id) | Transit Gateway Peering Attachment ID |
+<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
+## References
+
+- [cloudposse/terraform-aws-components](https://github.com/cloudposse/terraform-aws-components/tree/master/modules/TODO) - Cloud Posse's upstream component
```
Comment (nitrocode) [inline modules/tgw/cross-region-hub-connector/README.md:107]:
```suggestion
- [cloudposse/terraform-aws-components](https://github.com/cloudposse/terraform-aws-components/tree/master/modules/tgw/cross-region-hub-connector) - Cloud Posse's upstream component
```

File: modules/tgw/cross-region-hub-connector/variables.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
+variable "region" {
+  type        = string
+  description = "AWS Region"
+}
+
+variable "this_region" {
```
Comment (nitrocode) [inline modules/tgw/cross-region-hub-connector/variables.tf:6]: `this_` and `home_` were a bit confusing to me before and still confusing.

Could we simplify this before upstreaming ?

Comment (Benbentwo) [reply modules/tgw/cross-region-hub-connector/variables.tf:6]: I think `this` could be `current` but IMO `this` makes the most sense, home could be maybe `default`

File: modules/github-runners/README.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ To generate the token, please follow [these instructions](https://cloudposse.atl
 | <a name="input_region"></a> [region](#input\_region) | AWS Region | `string` | n/a | yes |
 | <a name="input_runner_labels"></a> [runner\_labels](#input\_runner\_labels) | List of labels to add to the GitHub Runner (e.g. 'Amazon Linux 2'). | `list(string)` | `[]` | no |
 | <a name="input_runner_role_additional_policy_arns"></a> [runner\_role\_additional\_policy\_arns](#input\_runner\_role\_additional\_policy\_arns) | List of policy ARNs that will be attached to the runners' default role on creation in addition to the defaults | `list(string)` | `[]` | no |
-| <a name="input_runner_version"></a> [runner\_version](#input\_runner\_version) | GitHub runner release version | `string` | `"2.287.1"` | no |
```
Comment (milldr) [inline modules/github-runners/README.md:149]: remove this file from this PR

**Review by milldr — APPROVED** [review]

**Pull request merged**